modules/inflate_geometry.py

```diff
@@ -0,0 +1,98 @@
+"""Functions for manipulating polygon geometry"""
+
+import math
+import numpy as np
+
+
+def inflate_polygon(vertices: np.ndarray, scale_distance: int) -> np.ndarray:
```

Comment:
can this be a list, makes it easier for users of the class

Reply:
both input and output, mb should have clarified. you can use the waypoint class that we have in modules because that one has both lonlat and altitude